src/scripts/migrate.ts

```diff
@@ -1,17 +1,19 @@
 import { getLogger } from '../logger';
-import { migrate } from '../database';
+import { db, migrate } from '../database';
 
 const logger = getLogger(__filename);
 
 logger.info('Starting migrations...');
 migrate()
-	.then(() => {
+	.then(async () => {
 		logger.info('Migrations complete.');
+		await db.close();
 	})
-	.catch((reason: unknown) => {
+	.catch(async (reason: unknown) => {
 		logger.error('Migrations failed!');
 		if (reason instanceof Error) {
 			logger.error(`${reason.message}`);
 		}
+		await db.close();
 		process.exit(1);
```

Comment:
Keeping this because I think we should exit with a `1` status code if migrations fail.

Reply:
We probably could re-throw here or something but this is less of a problem because we're already in an error state so we don't care.